jaffle_shop_online/models/marketing/marketing.yml

```
@@ -0,0 +1,40 @@
+        tests:
+          - positive_value
+      - name: attribution_revenue
+        tests:
+          - positive_value
+      - name: date_month
+        tests:
+          - dbt_utils.date_between:
+              min_date: '2020-01-01'  # Adjust this date as needed
+              max_date: "{{ '{{' }}current_timestamp() - 1 {{ '}}' }}"
+    
+    tests:
```

YAMLlint (1.37.1)

[error] duplication of key "tests" in mapping

(key-duplicates)